doc/platforms/srf06-cc26xx.md

```diff
@@ -219,7 +219,7 @@ For more information on the serial bootloader, see its README under the `tools/c
 For deployment/production images, it is _strongly_ recommended to:
 
 * Disable the ROM bootloader by defining `ROM_BOOTLOADER_ENABLE` as 0. In doing so, it is impossible to enter bootloader mode unless one first erases the device's flash.
```

Comment:
I'm thinking that references to ROM_BOOTLOADER_ENABLE (the backwards-compatible name) probably should be changed to `CCFG_CONF_ROM_BOOTLOADER_ENABLE`, for consistency. But that's maybe outside the scope of this PR